### DISCUSSION TITLE
Code Signing with Development Team - Unable to use School Apple ID. How does one remove the need of code signing?

### DISCUSSION THREAD

**Discussion by thamEnJie** [sst-inc/help#48] [Q&A - iOS] [answered]
Hi,

May I know how to get rid of this code signing error, if possible? I have tried using my apple id and adding it as a development team, but I think because of this year's school apple ID thing, we cannot use it as a development team. Much Appreciated!

![image](https://user-images.githubusercontent.com/89586587/137894684-27edf6cc-d20d-4a94-af12-ef43ad65e5ce.png)

has been really annoying cos I can't run my app...

**Comment by granwyntan** [comment, marked as answer]
Hi @thamEnJie, you might want to select **None** for the Development Team
<img width="295" alt="Screenshot 2021-10-19 at 18 52 22" src="https://user-images.githubusercontent.com/61010627/137895443-54c35843-bdb1-4979-93e1-7875cbb1ad06.png">

**Reply by thamEnJie** [reply]
As you can see in the next message I sent, it was already set to 'None'.




**The below is before being edited.**

> Hi @granwyntan, the only option for me is 'Other', and when I input 'None' It gives me the error that it is not a development team.
> 
> It also gives me another error of profiling.
> 
> <img width="501" alt="Screenshot" src="https://user-images.githubusercontent.com/89586587/137896906-ff6ac6e1-40ae-442c-8a18-ccefbc66a9e9.png">
> 
> 
> Much Appreciated!
> 
> (sorry, I forgot to @ you when I sent this just now)

**Reply by Joeispro99** [reply]
@thamEnJie what errors did Xcode give when you tried to use your apple id as a development team?

**Reply by thamEnJie** [reply]
Hi @Joeispro99!

So this is there error it gives me when I try to use my apple ID as a Development Team. (I suspect it is also linked to the restriction of not being able to download code using this apple id)
![image](https://user-images.githubusercontent.com/89586587/138590581-480b6726-ee6f-4fad-a83b-bc3d69666ee8.png)

As to what @granwyntan said earlier, it actually has already been set to 'None', I just looked at the wrong setting I think... 😬

**Reply by ghost** [reply]
Also, may I know if we are supposed to finish this by tomorrow? Cause this bug can't let me continue to code this app. I mean it is done, but I am just adding stuff.

**Reply by ghost** [reply]
Hi!

Ayaan from S1-03 has helped me. Apparently, somehow, my device was changed to 'My Mac' instead of the iPhone, not sure why...

**Reply by granwyntan** [reply]
Oh, I see, great that you have fixed it
